dashboard/modules/event/event_head.py

```diff
@@ -24,6 +24,8 @@
 JobEvents = OrderedDict
 dashboard_utils._json_compatible_types.add(JobEvents)
 
+MAX_EVENTS_TO_CACHE = int(os.environ.get("RAY_DASHBOARD_MAX_EVENTS_TO_CACHE", 10000))
```

Comment:
Is there a place it makes sense to add this to the docs? Some users might run memory growth tests and observe constant growth in memory, and be wondering whether it will continue indefinitely or stop at some point. If there's a place in the doc containing env vars they can set to verify that the memory growth indeed stops (`RAY_DASHBOARD_MAX_EVENTS_TO_CACHE`, `RAY_DASHBOARD_MAX_ACTORS_TO_CACHE`, etc) it could be useful.

Reply:
added as a follow up task